test/unit/transloadit/test_assembly.rb

```diff
@@ -142,14 +142,14 @@
       end
     end
 
-    describe 'when using the "submit!" method' do
+    describe 'when using the "create!" method' do
       it "must call the create! method with the same parameters" do
         VCR.use_cassette "submit_assembly" do
           file = open("lib/transloadit/version.rb")
           mocker = Minitest::Mock.new
           mocker.expect :call, nil, [file]
           @assembly.stub :create!, mocker do
-            @assembly.submit!(file)
+            @assembly.create!(file)
```

Comment:
IIRC this test ensured that the deprecated submit! method calls create!. By replacing the submit! call with a create! call, the test is now useless. Either we keep the test as is or we remove it entirely.

Reply:
Okay i see. 

I thought the output was smelly, reverted this 👌